src/execution/operator/filter/physical_filter.cpp

```diff
@@ -4,20 +4,22 @@
 #include "duckdb/parallel/thread_context.hpp"
 namespace duckdb {
 
-PhysicalFilter::PhysicalFilter(vector<LogicalType> types, vector<unique_ptr<Expression>> select_list,
-                               idx_t estimated_cardinality)
-    : CachingPhysicalOperator(PhysicalOperatorType::FILTER, std::move(types), estimated_cardinality) {
-	D_ASSERT(select_list.size() > 0);
-	if (select_list.size() > 1) {
-		// create a big AND out of the expressions
-		auto conjunction = make_uniq<BoundConjunctionExpression>(ExpressionType::CONJUNCTION_AND);
-		for (auto &expr : select_list) {
-			conjunction->children.push_back(std::move(expr));
-		}
-		expression = std::move(conjunction);
-	} else {
+PhysicalFilter::PhysicalFilter(ArenaAllocator &arena, vector<LogicalType> types,
```

Comment:
Instead of passing in the arena to all of these operators - can we pass in the child operator instead, and then use the arena from the child operator?